app/widgets/trackProcessing/trackProcessingWidget.php

```diff
@@ -77,15 +77,23 @@ public function renderWidget($ps_widget_id, &$pa_settings) {
 			$this->opo_view->setVar('hours', $pa_settings['hours']);
 			
 			$vo_tq = new TaskQueue();
+			$qr_tq_total_count = $this->opo_db->query("
+				SELECT count(*) as _count
+				FROM ca_task_queue ctq 
+				WHERE ctq.completed_on > ?
+			", time() - (60*60*$pa_settings['hours']));
+
 			$qr_completed = $this->opo_db->query("
 				SELECT tq.*, u.fname, u.lname 
```

Comment:
@kehh this is pulling in _all_ task queue fields (`tq.*`), which the notes field for SolrIndexer in my local is around 3MB each. 

Selecting the specific fields that are used would be a lot more efficient, however I think limiting the query itself also makes sense as PHP needs to load the result of the whole query to be able to loop though it anyway, so I don't know if we care to pursue additional optimisations.